src/lavinmqctl.cr

```diff
@@ -255,14 +261,56 @@ class LavinMQCtl
   private def handle_response(resp, *ok)
     return if ok.includes? resp.status_code
     if resp.status_code == 503
-      puts "[ERROR] #{resp.body}"
+      output resp.body
       exit 2
     end
-    puts "#{resp.status_code} - #{resp.status}"
-    puts resp.body if resp.body? && !resp.headers["Content-Type"]?.try(&.starts_with?("text/html"))
+    output "#{resp.status_code} - #{resp.status}"
+    output resp.body if resp.body? && !resp.headers["Content-Type"]?.try(&.starts_with?("text/html"))
     exit 1
   end
 
+  private def output(data, columns = nil)
+    if @options["format"]? == "json"
+      data.to_json(STDOUT)
+      puts
+    else
+      case data
+      when Hash, NamedTuple
+        data.keys.each do |k|
+          puts "#{k}: #{data[k]}"
+        end
+      when Array
+        output_array(data, columns)
+      else
+        puts data
+      end
+    end
+  end
+
+  private def output_array(data : Array, columns : Array(String)?)
+    if columns
+      puts columns.join("\t")
+    else
+      if first = data.first?
+        first = first.as_h if first.is_a?(JSON::Any)
+        puts first.keys.join("\t")
+      end
+    end
+    data.each do |item|
+      values = [] of String
+      if item.is_a?(Hash) || item.is_a?(NamedTuple)
+        values = item.values.map &.to_s
```

Comment:
```suggestion
        puts item.each_value.join(STDOUT, "\t")
```

Reply:
`each_value` expects to be invoked with a block, so this throws an error. Maybe just print each value instead of the join? 
```
    data.each do |item|
      if item.is_a?(Hash) || item.is_a?(NamedTuple)
        item.each_value { |v| print "#{v}\t" }
      elsif item.is_a?(JSON::Any)
        if hash = item.as_h
          hash.each_value { |v| print "#{v}\t" }
        end
      else
        print item.to_s
      end
      puts
    end
```

Reply:
You can do like this:
```crystal
data.each do |item|
  case item
  when Hash
    item.each_value.join(STDOUT, "\t")
  when JSON::Any
    item.as_h.each_value.join(STDOUT, "\t")
  when NamedTuple
    item.keys.each.join(STDOUT, "\t")
  else
    item.to_s(STDOUT)
  end
  puts
end```

`NamedTuple#keys` returns a tuple, so no allocation.

Reply:
nice 👍 (but we want the values from the `NamedTuple` so we should use `NamedTuple#values` instead I guess. Also returns a tuple)

Reply:
Oh, right! 😁

Reply:
Btw, maybe you can change `item.keys.each.join(STDOUT, "\t")` to just `item.keys.join(STDOUT, "\t")`?